assets/js/components/settings/SettingsActiveModule/Header.js

```diff
@@ -246,63 +284,32 @@ export default function Header( { slug } ) {
 						alignMiddle
 						mdAlignRight
 					>
-						{ connected &&
-							( slug !== 'analytics' || isGA4Connected ) && (
-								<p className="googlesitekit-settings-module__status">
-									{ __( 'Connected', 'google-site-kit' ) }
-
-									<span className="googlesitekit-settings-module__status-icon googlesitekit-settings-module__status-icon--connected" />
-								</p>
-							) }
-
-						{ connected &&
-							slug === 'analytics' &&
-							hasAnalyticsAccess &&
-							! isGA4Connected && (
-								<Fragment>
-									<Button
-										onClick={ handleConnectGA4ButtonClick }
-									>
-										{ __(
-											'Connect Google Analytics 4',
-											'google-site-kit'
-										) }
-									</Button>
-									<span className="googlesitekit-settings-module__status-icon googlesitekit-settings-module__status-icon--not-connected" />
-								</Fragment>
-							) }
-
-						{ connected &&
-							slug === 'analytics' &&
-							! hasAnalyticsAccess &&
-							! isGA4Connected && (
-								<p className="googlesitekit-settings-module__status">
-									{ __(
-										'Google Analytics 4 is not connected',
-										'google-site-kit'
-									) }
-									<span className="googlesitekit-settings-module__status-icon googlesitekit-settings-module__status-icon--not-connected" />
-								</p>
+						<div
+							className={ classnames(
+								'googlesitekit-settings-module__status',
+								`googlesitekit-settings-module__status--${
+									showAsConnected
+										? 'connected'
+										: 'not-connected'
+								}`,
```

Comment:
I would suggest having these as two separate conditional classes with opposite logic as it makes it easier to find the usage when searching the codebase.

Reply:
Very valid suggestion, updated, thank you!